include/flang/Parser/dump-parse-tree.h

```diff
@@ -761,7 +760,8 @@ class ParseTreeDumper {
 protected:
   // Return a Fortran representation of this node to include in the dump
   template<typename T> std::string AsFortran(const T &x) {
-    std::ostringstream ss;
+    std::string buf;
+    llvm::raw_string_ostream ss {buf};
```

Comment:
Is there a formatting issue here?

Reply:
What's wrong with this formatting?

Reply:
Thre is a space between ss and the brace.

Reply:
I did not run clang-format-7 on the header files. Only on the cpp.